final-project/src/tier1/action_primitives.py

```diff
@@ -64,6 +66,8 @@ def curved_line_rotations(self):
                 print("matrix" + str(i+10))
                 print(matrix[i])
         # Convert to bezier curve
+        matrix[matrix<-1]=-1
+        matrix[matrix>1]=1
```

Comment:
Again not sure the point of the 1.425 and these

Reply:
Clipping last end point to either 1.0 or -1.0